Content.Shared/Pulling/Systems/SharedPullingSystem.Actions.cs

```diff
@@ -123,6 +128,12 @@ public bool TryStartPull(SharedPullerComponent puller, SharedPullableComponent p
                 return false;
             }
 
+            if (pullablePhysics.FixedRotation)
+            {
+                pullable.HasFixedRot = true;
+                pullablePhysics.FixedRotation = false;
+            }
+
```

Comment:
Wouldn't you just do this at the end of the method when it returns true?

Reply:
I could move it there, I just thought it would work better grouped with the checks.